tools/wasm-testcompile/src/lib.rs

```diff
@@ -1,14 +1,33 @@
 use sspi::internal::credssp;
+use sspi::negotiate::network_client::{NetworkClient, NetworkClientFactory};
 use sspi::ntlm::AuthIdentity;
 use wasm_bindgen::prelude::*;
 
+#[derive(Debug, Clone)]
+struct DumpNetworkClientFactory;
```

Comment:
I’m unsure of the name. Do you mean "Dummy" here?

```suggestion
struct DummyNetworkClientFactory;
```

Reply:
Yeah 😄

Reply:
Fixed